grub/02_ignition_firstboot

```diff
@@ -9,5 +9,16 @@ search --set=bootpart --label boot
 # to be used later on the kernel command line.
 set ignition_firstboot=""
 if [ -f "(${bootpart})/ignition.firstboot" ]; then
-    set ignition_firstboot="ignition.firstboot rd.neednet=1 ip=dhcp"
+    # default to dhcp networking parameters to be used with ignition 
+    ignition_network_kcmdline='rd.neednet=1 ip=dhcp'
```

Comment:
I did test this and it worked but I'm going to update it to be `set ignition_network_kcmdline` instead. will push up change after I test it.